### PR TITLE
Update 2D UNet Arabidopsis Apical Stem Cells

### PR DESCRIPTION
**Pull request by bioimageiobot**
This is an automatic PR created by the @bioimageiobot regarding changes to the resource item 10.5281/zenodo.6334881.
The following version(s) will be added:
  - [7781054 (version 7)](https://www.doi.org/10.5281/zenodo.7781054)

Please review the changes and make sure the new item or version(s) pass the following check list:
- [ ] Passed the BioImage.IO CI tests: static (and dynamic) validations
- [ ] The meta information for the RDF item is complete
  - [ ] The tags are complete and describe the model
  - [ ] Naming is intuitive and descriptive, example:  Multi-Organ Nucleus Segmentation (StarDist 2D)
  - [ ] Authors are provided
  - [ ] Documentation is complete
    * For models, include an overview, describe how the model is trained, what is the training data, how to use the model, how to validate the results and list the references. TODO: Model documentation template.
- [ ] Approved by at least one of the BioImage.IO admin team member.

Maintainers: none specified

Note: If you updated or re-uploaded another version for the current item on Zenodo, this PR won't be changed automatically. To proceed, you can do the following:
 1. Block this version, but keep looking for future versions: Edit the current resource.yaml and keep the top-level `status` field as `accepted`, but change the `status` under the current version to `blocked`.
 2. Accept this version and keep looking for future versions: Merge this PR for now.
 3. Keep proposed version(s) (and this resource in general if it is new) as pending: Close this PR without merging.

 Then wait for the CI on the main branch to complete. It should detect the new version(s) and create another PR for the new version(s).

Previous PRs of this resource: https://github.com/bioimage-io/collection-bioimage-io/pull/330, https://github.com/bioimage-io/collection-bioimage-io/pull/373, https://github.com/bioimage-io/collection-bioimage-io/pull/554, https://github.com/bioimage-io/collection-bioimage-io/pull/565, https://github.com/bioimage-io/collection-bioimage-io/pull/579, https://github.com/bioimage-io/collection-bioimage-io/pull/584

### CODE REVIEW

**Comment by FynnBe** [comment]
@wolny, no torchscript weights for this model?

**Comment by wolny** [comment]
ok I'm a bit at loss with the torchscript business... If you click the zenodo link above (https://www.doi.org/10.5281/zenodo.7781054), you can see that there are torchscript weights associated uploaded. You say that there are no weights, so I assume that the torchscript weights I've uploaded are somehow not reflected in bioimage.io? What the proper procedure to include them then?

**Comment by FynnBe** [comment]
they are not referenced in the [rdf.yaml](https://zenodo.org/record/7781054/files/rdf.yaml?download=1)

**Comment by FynnBe** [comment]
I can fix this here in the PR...

**Comment by wolny** [comment]
thanks @FynnBe! Could you please do that in the other PRs as well (the story is similar in there): https://github.com/bioimage-io/collection-bioimage-io/pull/587 https://github.com/bioimage-io/collection-bioimage-io/pull/588

**Comment by FynnBe** [comment]
the torchscript weights do not reproduce the test outputs:
```
'error': 'Output and expected output disagree:\n'
            ' \n'
            'Arrays are not almost equal to 4 decimals\n'
            '\n'
            'Mismatched elements: 262140 / 262144 (100%)\n'
            'Max absolute difference: 21.93094\n'
            'Max relative difference: 7.337398e+10\n'
            ' x: array([[[[ -5.9214,  -7.8174,  -8.0128, ...,  -8.4763,  -7.6826,\n'
            '           -5.4946],\n'
            '         [ -7.8[42](https://github.com/bioimage-io/collection-bioimage-io/actions/runs/4555392889/jobs/8034508770?pr=586#step:9:43)2, -10.014 , -10.0676, ..., -11.5718, -10.8581,...\n'
            ' y: array([[[[2.67[43](https://github.com/bioimage-io/collection-bioimage-io/actions/runs/4555392889/jobs/8034508770?pr=586#step:9:44)e-03, 4.0252e-04, 3.3108e-04, ..., 2.0831e-04,\n'
            '          4.6058e-04, 4.0922e-03],\n'
            '         [3.9265e-04, 4.[47](https://github.com/bioimage-io/collection-bioimage-io/actions/runs/4555392889/jobs/8034508770?pr=586#step:9:48)66e-05, 4.2430e-05, ..., 9.4277e-06,...',
```
I'll block this version

**Comment by wolny** [comment]
thanks for looking into this @FynnBe. I think the problem is the same as before, i.e. I didn't put the model into eval mode before exporting the torchscript weights.
I just noticed that there is an example in the notebook of how to add torchscript weights to the `model.zip` (https://github.com/bioimage-io/core-bioimage-io-python/blob/main/example/model_creation.ipynb). I guess the best course of action would be for me to export the correct torchscript weights, add them to the zip and reupload via bioimage.io, right?

**Comment by FynnBe** [comment]
> I guess the best course of action would be for me to export the correct torchscript weights, add them to the zip and reupload via bioimage.io, right?

that or directly editing the record on zenodo would work as well. just make sure to add the `torchscript` entry in the rdf.yaml as well...
